.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I don't think we should assume people have npx locally to run it.
Project runs via dotrun, so it's not necessary to have local node.

Also I guess I would prefer to run simply the built-in lint commands (like `dotrun test` in case of Vanilla) rather than having a separate config for linting only the staged files.